source/content/guides/decoupled-sites/03-starter-kits.md

```diff
@@ -16,35 +16,60 @@ reviewed: "2022-08-11"
 * [WordPress and Gatsby](https://github.com/pantheon-systems/decoupled-wordpress-recommended) - This configuration supports static site generation. 
 
 
 ## Backend Installation and Configuration
 
+Choose from the following approaches to configure your backend:
+
+* Build Tools:
+  * Use this option if testing is important to your team's workflow. With Build Tools you will not have to manually push changes to your code repository.
+
+* Dashboard Upstream: 
+  *  Use this option if you prefer a more streamlined configuration. 
+
+
 <TabList>
 
 <Tab title="Drupal Backend" id="drupal-install" active={true}>
 
+#### Dashboard Upstream Install
+
+You can install the Drupal backend by using the Dashboard upstream.
+
+Navigate to the Pantheon Dashboard to create a site from the [Decoupled Drupal Composer Managed upstream](https://dashboard.pantheon.io/sites/create?upstream_id=c76c0e51-ad85-41d7-b095-a98a75869760) 
+
+Alternatively, you can use Terminus. Enter the following command to create a site with the Decoupled Drupal Composer Managed upstream.
+
+```
+  terminus site:create my-new-site "Describe Site" empty --org='My Team Name' c76c0e51-ad85-41d7-b095-a98a75869760
+```
+  * Replace `{My Team Name}` with your team name, for example `My Agency`. This can also be omitted.
+  * Note that `c76c0e51-ad85-41d7-b095-a98a75869760` is the `upstream_id` for Decoupled Drupal Composer Managed.
+
+
+#### Installing Using Build Tools
+
 - Run `terminus build:project:create`:
 
   ```
   terminus build:project:create \
-    --team='{My Team Name}' \
-    --template-repository="git@github.com:pantheon-systems/decoupled-drupal-recommended.git" pantheon-systems/decoupled-drupal-recommended \
-    --ci-template='git@github.com:pantheon-systems/advanced-ci-templates' \
-    --visibility private {PROJECT_NAME} \
-    --stability=dev \
-    --profile="pantheon_decoupled_profile"
+  --team='{My Team Name}' \
+  --template-repository="git@github.com:pantheon-upstreams/decoupled-drupal-composer-managed.git" pantheon-upstreams/decoupled-drupal-composer-managed \
+  --visibility private {PROJECT_NAME} \
+  --profile="pantheon_decoupled_profile" \
+  --stability=dev
   ```
 
   * Replace `{PROJECT_NAME}` with your project name, for example `decoupled-drupal`.
 
   * Replace `'{My Team Name}'` with your team name, for example `My Agency`. This can also be omitted.
 
-This action will result in a GitHub repository created for this new codebase under the authenticated user's namespace (unless the `--org` option is used), a site created on Pantheon and a CircleCI project created for automated deployments.
+This action will result in a GitHub repository created for this new codebase under the authenticated user's namespace (unless the `--org` option is used), a site created on Pantheon, and a CircleCI project created for automated deployments.
 
```

Comment:
Would it be worthwhile to add a line like the following that links to additional build tools options, if someone wanted to use another CI provider for example?

For additional options on various repository or CI providers please refer to [Commands are available as part of the Build Tools plugin](https://github.com/pantheon-systems/terminus-build-tools-plugin#commands)